modules/reference/pages/cluster-properties.adoc

```diff
@@ -3151,7 +3138,7 @@ Interval for metadata dissemination batching.
 
 === metadata_dissemination_retries
 
-Number of attempts of looking up a topic's meta data like shard before failing a request.
+Number of attempts to look up a topic's metadata-like shard before a request fails.
```

Comment:
ENG: Please confirm whether this edit retains the meaning of the property description